### PR TITLE
[FEATURE] Add migration to associative items array keys

### PR DESCRIPTION
**Pull request by nhovratov**
Fixes: #3399

### CODE REVIEW

**Comment by sabbelasichon** [comment]
@nhovratov Thank you very much. Awesome to see new people contributing to the project. I will have a look soon.

File: tests/Rector/v12/v3/tca/MigrateItemsIndexedKeysToAssociativeRector/Fixture/fixture.php.inc [hunk 1]
```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace Ssch\TYPO3Rector\Tests\Rector\v12\v0\tca\MigrateItemsIndexedKeysToAssociativeRector\Fixture;
```
Comment (helsner) [inline tests/Rector/v12/v3/tca/MigrateItemsIndexedKeysToAssociativeRector/Fixture/fixture.php.inc:3]: namespace here is not correct

**Comment by kevin-appelt** [comment]
I guess this could be the most wanted rector rule for TYPO3 v12 upgrades.
Anything I can do to get this merged? :)

**Comment by helsner** [comment]
@sabbelasichon can you check along?
are there any reasons to not merge and freshly release?

**Pull request merged**